### PR TITLE
Use all YAML files for Jenkins in credentials repo

### PR DESCRIPTION
**Pull request by lfdebrux**
Allows credentials files added by https://github.com/alphagov/digitalmarketplace-credentials/pull/221 to be picked up by Ansible.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,9 @@ jenkins: requirements ## Run Jenkins playbook
 	JENKINS_VARS_FILE=$$(mktemp) ;\
 	PRIVATE_KEY_FILE=$$(mktemp) ;\
 	trap 'rm $$JENKINS_VARS_FILE $$PRIVATE_KEY_FILE' EXIT ;\
-	${DM_CREDENTIALS_REPO}/sops-wrapper -d ${DM_CREDENTIALS_REPO}/jenkins-vars/jenkins.yaml > $$JENKINS_VARS_FILE ;\
+	for varfile in ${DM_CREDENTIALS_REPO}/jenkins-vars/*.yaml; do \
```
Comment (katstevens) [inline Makefile]: Should we allow `.yml` as well as `yaml` for flexibility, or should we enforce consistency? (I am on the fence...)

Comment (lfdebrux) [reply Makefile]: Google says... https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml?rq=1

Comment (lfdebrux) [reply Makefile]: I prefer consistency

Comment (lfdebrux) [reply Makefile]: Also the only yaml files in the relevant folder so far are .yaml so hopefully that means that we shouldn't need the flexibility

Comment (katstevens) [reply Makefile]: Sounds good to me!

**Pull request merged**